app/user/user_exceptions.c

```diff
@@ -73,6 +75,9 @@ void load_non_32_wide_handler (struct exception_frame *ef, uint32_t cause)
     /* Turns out we couldn't fix this, trigger a system break instead
      * and hang if the break doesn't get handled. This is effectively
      * what would happen if the default handler was installed. */
+    if (load_store_handler) {
```

Comment:
Before I merge this, could I suggest you either update the comment to mention the chaining, or move the chaining above the comment?